StackExchange.Redis/StackExchange/Redis/GeoUnit.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+
+namespace StackExchange.Redis
+{
+    /// <summary>
+    /// Units associated with Geo Commands
+    /// </summary>
+    public enum GeoUnit : short
+    {
+        /// <summary>
+        /// Meters
+        /// </summary>
+        m,
```

Comment:
Part of me is shouting that this should be `Meters`, `Kilometers`, `Miles`, etc... thoughts?